com.chartboost.mediation/Runtime/Banner/ChartboostMediationBannerAdSize.cs

```diff
@@ -41,49 +44,49 @@ private ChartboostMediationBannerAdSize(string name, float width, float height,
 
         public static ChartboostMediationBannerAdSize Adaptive(float width)
         {
-            return new ChartboostMediationBannerAdSize("ADAPTIVE", width, 0);
+            return new ChartboostMediationBannerAdSize(ChartboostMediationBannerName.Adaptive, width, 0);
         }
 
         public static ChartboostMediationBannerAdSize Adaptive(float width, float height)
         {
-            return new ChartboostMediationBannerAdSize("ADAPTIVE", width, height);
+            return new ChartboostMediationBannerAdSize(ChartboostMediationBannerName.Adaptive, width, height);
         }
 
         #region static conveniences
-        public static readonly ChartboostMediationBannerAdSize Standard = new ChartboostMediationBannerAdSize("STANDARD", BannerSize.STANDARD.Item1, BannerSize.STANDARD.Item2, ChartboostMediationBannerType.Fixed);
+        public static readonly ChartboostMediationBannerAdSize Standard = new ChartboostMediationBannerAdSize(ChartboostMediationBannerName.Standard, BannerSize.STANDARD.Item1, BannerSize.STANDARD.Item2, ChartboostMediationBannerType.Fixed);
 
-        public static readonly ChartboostMediationBannerAdSize MediumRect = new ChartboostMediationBannerAdSize("MEDIUM", BannerSize.MEDIUM.Item1, BannerSize.MEDIUM.Item2, ChartboostMediationBannerType.Fixed);
+        public static readonly ChartboostMediationBannerAdSize MediumRect = new ChartboostMediationBannerAdSize(ChartboostMediationBannerName.Medium, BannerSize.MEDIUM.Item1, BannerSize.MEDIUM.Item2, ChartboostMediationBannerType.Fixed);
```

Comment:
Should `BannerSize` be `ChartboostMediationBannerSize`? I also have the feeling we can combine this and `ChartboostMediationBannerName`, to make it easier to understand, since they refer kinda to the same thing.

Reply:
I'm thinking of doing couple of things 
1. Rename `ChartboostMediationBannerName` to `ChartboostMediationBannerSizeType` since that makes more sense now that it is no longer a string 
2. Add extension method to `ChartboostMediationBannerSizeType` that returns the size for the enum value so it would look something like 

```
public static readonly ChartboostMediationBannerAdSize MediumRect = new 
ChartboostMediationBannerAdSize(ChartboostMediationBannerSizeType.Medium, 
ChartboostMediationBannerSizeType.Medium.Size().x, 
ChartboostMediationBannerSizeType.Medium.Size().y, 
ChartboostMediationBannerType.Fixed);

Reply:
Yeah, this is more consistent and will make it easier to understand how they relate to each other. 👍🏻